### PR TITLE
Update CIIP consent

### PR DESCRIPTION
**Pull request by dimak1**

### CODE REVIEW

File: app/containers/Applications/ApplicationWizardConfirmation.tsx [hunk 1]
```diff
@@ -212,11 +212,15 @@ export const ApplicationWizardConfirmationComponent: React.FunctionComponent<Pro
       <br />
       {revision.certificationSignatureIsValid ? (
         <>
-          <h5>
-            Thank you for reviewing the application information. The Certifier
-            has signed off on this application. You may now submit the
-            application.
-          </h5>
+          <div>
+            “By submitting the application the applicant agrees that the
```
Comment (dleard) [inline app/containers/Applications/ApplicationWizardConfirmation.tsx]: Much better information than my placeholder text :+1: 
Could probably use some styling though, maybe throw it in a card component with a title like 'Attention:' or 'Before you Submit:'  & put the legalese in a Card.Text, I think there's some error notifications in ApplicationWizardConfirmation that use the card element you can probably snag & replace

File: app/containers/Applications/ApplicationWizardConfirmation.tsx [hunk 2]
```diff
@@ -212,11 +212,15 @@ export const ApplicationWizardConfirmationComponent: React.FunctionComponent<Pro
...
+            information contained on this application, or information contained
+            in emission reports under the Greenhouse Gas Industrial Reporting
+            and Control Act, may be disclosed to British Columbia government
+            employees, contractors and agencies for the purpose of administering
+            the CleanBC Program for Industry or the Greenhouse Gas Industrial
+            Reporting and Control Act”.
```
Comment (dleard) [inline app/containers/Applications/ApplicationWizardConfirmation.tsx]: Probably don't need the quotes around the text

Comment (dimak1) [reply app/containers/Applications/ApplicationWizardConfirmation.tsx]: Updated

File: app/containers/Applications/ApplicationWizardConfirmation.tsx (hunk 1 above)
Comment (dimak1) [reply app/containers/Applications/ApplicationWizardConfirmation.tsx]: Added Card

**Comment by dimak1** [comment]
Updated styling for consent @dleard 
consent

**Pull request merged**